SPECS/kernel/config

```diff
@@ -7278,7 +7286,7 @@ CONFIG_HARDENED_USERCOPY=y
 CONFIG_FORTIFY_SOURCE=y
 # CONFIG_STATIC_USERMODEHELPER is not set
 CONFIG_SECURITY_SELINUX=y
-# CONFIG_SECURITY_SELINUX_BOOTPARAM is not set
+CONFIG_SECURITY_SELINUX_BOOTPARAM=y
```

Comment:
Does this require `SECURITY_SELINUX_BOOTPARAM_VALUE` to be set as well?

Reply:
I do not see `SECURITY_SELINUX_BOOTPARAM_VALUE` in the code. We have only been setting `CONFIG_SECURITY_SELINUX_BOOTPARAM` to enable the kernel command line parameter. It allows images to be built with selinux built-in but disabled or in permissive mode and does not change functionality should the parameter not be used.

Reply:
Ah yup, looks like `SECURITY_SELINUX_BOOTPARAM_VALUE` option was removed. Sounds good :)